packages/react-composites/tests/browser/call/fixture.ts

```diff
@@ -52,8 +52,16 @@ export const test = base.extend<unknown, ChatWorkerFixtures>({
     /* eslint-disable-next-line no-empty-pattern */
     async ({}, use) => {
       const browser = await chromium.launch({
-        args: ['--start-maximized', '--disable-features=site-per-process'],
-        headless: false
+        executablePath: process.env.CHROME_PATH,
+        args: [
+          '--start-maximized',
+          '--disable-gpu',
+          '--use-fake-ui-for-media-stream',
+          '--use-fake-device-for-media-stream',
+          '--allow-file-access',
+          `--use-file-for-fake-video-capture=${path.join(__dirname, 'puppy.y4m')}`
```

Comment:
To help with file size, sounds like we can use mpjeg also? (if its more compressed?)
https://source.chromium.org/chromium/chromium/src/+/master:media/capture/video/file_video_capture_device.h;l=33

Reply:
If we use a super small file that is literally one frame, what happens? (does it just loop continuosuly?)

Reply:
From that link, is just JPEG supported?? 
![image](https://user-images.githubusercontent.com/2684369/126576251-fbfbc48e-7d69-4961-abfa-4a72bb7ac53e.png)

Reply:
Did try to use MJPEG and JPEG but it didn't work. I will try again today.

Reply:
So JPEG and MJPEG is not working.
But I found a .y4m file in chromium repository that's just 744kb and is small enough to be used in our tests.

Reply:
Sounds good to me!

Reply:
(assuming no licensing/copyright issues from out using theirs)

Reply:
It's just 4 colored squares :D